server/routes/api/v1/order.js

```diff
@@ -0,0 +1,7 @@
+var express = require("express");
+var router = express.Router();
+const { OrderService } = require("../../../services");
+
+router.post("/createNew", OrderService.createOrder);
```

Comment:
by the api convention, this should be `/orders` and not `/createNew` @RubyTamrakar